### ISSUE TITLE
Review annotations to GO:0090079 translation regulator activity, nucleic acid binding

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5564]
Dear all, 

The proposal has been made to obsolete GO:0090079 translation regulator activity, nucleic acid binding , and its regulation children, 
see https://github.com/geneontology/go-ontology/issues/29353

Experimental annotations that need to be reviewed are here: 
https://docs.google.com/spreadsheets/d/1lVwOAtSTM0z4lFuDbSTHNMeYa7qvi2mrYve-mYs5XZQ/edit?gid=0#gid=0

Impacted groups: 
MGI (1)

** Group contacts: https://github.com/geneontology/go-site/blob/master/metadata/group-contacts.csv

Thanks.

**Comment by ValWood** [comment]
- For some reason the spreadsheet is for *Every* child term rather than just the term in question. Is that normal?
- there is only a single direct annotation direct to this term
https://www.uniprot.org/uniprotkb/Q5SF07/entry
would we class this as a "translation regulator"?
It is regulating translation by seqestering RNA, but I would not make this "translation regulator" activity?
@pgaudet what do you think? this raises more questions which I will put into the main ticket

**Comment by deustp01** [comment]
Reactome is impacted and not on any list I can find. I have other ways to find the affected annotations but advice on appropriate replacement GO terms would be really useful.
Thanks
Peter

**Comment by ValWood** [comment]
Hi @deustp01 
I have not actioned this ticket yet because it requires further discussion, possibly on an ontology call.
https://github.com/geneontology/go-ontology/issues/29089
val

**Comment by pgaudet** [comment]
> For some reason the spreadsheet is for Every child term rather than just the term in question. Is that normal?

Yes, you picked the template 'Annotation review for obsoletion - term + regulation children'
to have only exact matches, pick the second option, 'Annotation review for obsoletion'

**Comment by pgaudet** [comment]
> there is only a single direct annotation direct to this term
https://www.uniprot.org/uniprotkb/Q5SF07/entry
would we class this as a "translation regulator"?
It is regulating translation by sequestering RNA, but I would not make this "translation regulator" activity?

How I would approach this is like you did for the RNA processing: first define clearly what is the process/pathway, and then we can more easily judge what's in out out. If you dont map out the plan first then steps will fall out that you hadn't thought about. 

WRT to https://www.uniprot.org/uniprotkb/Q5SF07/entry: 
it seems to correspond to '[mRNA localization resulting in post-transcriptional regulation of gene expression](http://purl.obolibrary.org/obo/GO_0010609)', although whether this is a process is a bit debatable. I would rather have some 'mRNA sequestering activity' involved in 'negative regulation of gene expression'; GO:0010609 doesn't indicate the direction of the regulation (+ or -). 

What do you think ?

**Comment by ValWood** [comment]
I thought most of the other terms would probably stay,  just need reorganizing to separate translation factors from translation regulators.
We already decided this one should go in the main ticket, presumably because it was 'nucleic acid' rather than 'RNA' so it was an unnecessary grouping term here:
see https://github.com/geneontology/go-ontology/issues/29089
(I now realise I linked to the wrong ticket at the top)

Removing this and rehousing the children directly under the regulator was the first obvious step.

**Comment by ValWood** [comment]
Actually the correct ticket is linked from the ontology ticket.
The approach is described there.
This ticket corresponds to "step 1" here
https://github.com/geneontology/go-ontology/issues/29089#issuecomment-2447773890
which you 👍

**Comment by ValWood** [comment]
because this one is described as "unnecessary grouping term" I expected the annotation would move to one of the children
[GO:0000900](https://www.ebi.ac.uk/QuickGO/term/GO:0000900)    mRNA regulatory element binding translation repressor activity
for example.
This ticket is just to remove  the term because it is "nucleic acid" and all of the translation regulators should be RNA?

**Comment by ValWood** [comment]
Hi @deustp01 in answer to your question, you should be able to move to one of the existing children (which will remain)

**Comment by LiNiMGI** [comment]
MGI done.
